arch/arm/core/aarch32/cortex_m/mpu/arm_core_mpu.c

```diff
@@ -148,6 +148,20 @@ void z_arm_configure_dynamic_mpu_regions(struct k_thread *thread)
 	 * of the respective dynamic MPU regions to be programmed for
 	 * the given thread. The array of partitions (along with its
 	 * actual size) will be supplied to the underlying MPU driver.
+	 *
+	 * The drivers of what regions get configured are CONFIG_USERSPACE,
+	 * CONFIG_MPU_STACK_GUARD, and K_USER/supervisor threads.
+	 *
+	 * If CONFIG_USERSPACE then any Memory Domains associated with the
```

Comment:
better: CONFIG_USERSPACE=y, and adapt this, also, in more occurrences in the text
Alternatively: "if CONFIG_USERSPACE is defined..."

Reply:
Also: each thread is member of a single domain, so the correct wording is:
`...any application memory partitions associated with the thread get a defined region`

Reply:
ok